### PR TITLE
[DOCFIX] Presto doc update

### PR DESCRIPTION
**Pull request by yupeng9**
Update the Hive and Presto docs on the instructions of alluxio properties setting.

### CODE REVIEW

**Comment by yupeng9** [comment]
Jenkins, test this please

**Comment by yupeng9** [comment]
Jenkins, test this please

File: docs/en/Running-Presto-with-Alluxio.md [hunk 1]
```diff
@@ -68,6 +68,13 @@ Similar to above, add additional Alluxio properties to `core-site.xml` of Hadoop
 </property>
 ```
 
+Alternatively, you can also append the path to [`alluxio-site.properties`](Configuration-Settings.html) to Presto's JVM config at `etc/jvm.config` under Presto folder. The advantage of this approach is to have all the Alluxio properties set within the same file of `alluxio-site.properties`.
+
+```bash
+...
+-Xbootclasspath/p:<path-to-alluxio-site-properties>
```
Comment (gpang) [inline docs/en/Running-Presto-with-Alluxio.md]: Does it matter if it uses `/a` or `/p`?

Comment (yupeng9) [inline docs/en/Running-Presto-with-Alluxio.md]: I don't think it matters, so I picked either one.

**Pull request merged**